assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.js

```diff
@@ -140,6 +165,23 @@ export default function SettingsEdit() {
 						) }
 					/>
 				) }
+
+				{ isRRMv2Enabled &&
+					hasModuleAccess &&
+					publicationAvailable &&
+					isCurrentProductMissing && (
```

Comment:
Assuming we refactor as proposed above:

```suggestion
					missingProductID && (
```